### PR TITLE
Docs: rearranging plugin guides.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: rearranging plugin guides. Closes cksource/ckeditor5-commercial#6120

---

### Additional information

_Merge with https://github.com/cksource/ckeditor5-commercial/pull/6122_

### CODE REVIEW

File: docs/updating/update-to-33.md
Comment (gorzelinski) [inline docs/updating/update-to-33.md:1]: Do we want to change links retrospectively?

**Review by gorzelinski — COMMENTED** [review]
Everything seems to work fine - no errors in links. I just added one question.

File: docs/updating/update-to-33.md
Comment (godai78) [reply docs/updating/update-to-33.md:1]: I'll check this one. Most of the times we should not.

Comment (godai78) [reply docs/updating/update-to-33.md:1]: We have to, in this case, there is no legacy version of this guide. Or we may delink it.

**Pull request merged**